src/main.h

```diff
@@ -52,6 +52,17 @@ CommandLineOptions;
 extern CommandLineOptions cl_options;
 
 
+/* Information about command line entries that can not be stored in GOptionEntry. */
+typedef struct
+{
+	gboolean persistent;  /* The option should be passed to "New Window" */
+}
+GeanyOptionEntryAux;
+
+extern GOptionEntry optentries[];
+extern GeanyOptionEntryAux optentries_aux[];
```

Comment:
I'm not very fond of exposing those outside of main, maybe it'd be nice to find a clean way to encapsulate access to those… any idea?